lib/app/modules/Home/views/local_widgets/home_character_hp_xp_view.dart

```diff
@@ -1,40 +1,108 @@
+          onTap: () => Get.dialog(const HPDialog()),
+          child: const Padding(
+            padding: EdgeInsets.all(4),
+            child: HpBar(),
+          ),
+        ),
+      ),
+      const SizedBox(width: 16),
+      const Expanded(
+        child: _EXP(),
+      ),
+    ],
+  );
+}
+
+class _EXP extends StatefulWidget {
+  const _EXP();
 
+  @override
+  State<_EXP> createState() => _EXPState();
+}
+
+class _EXPState extends State<_EXP> with CharacterServiceMixin {
+  var _hidden = true;
+
+  Timer _timer = Timer(Duration.zero, () {});
+  
+  void showButtons({bool shown = true}) {
+    setState(() {_hidden = !shown;});
+    _timer.cancel();
+    if (!shown) return;
+    _timer = Timer(const Duration(seconds: 2), () {
+      setState(() {_hidden = true;});
+    });
+  }
+  
   @override
   Widget build(BuildContext context) {
-    return Row(
+    return Stack(
       children: [
-        Expanded(
-          child: InkWell(
-            splashColor: Theme.of(context).splashColor,
-            child: const Padding(
-              padding: EdgeInsets.all(4),
-              child: HpBar(),
-            ),
-            borderRadius: BorderRadius.circular(10),
-            onTap: () => Get.dialog(const HPDialog()),
+        InkWell(
+          splashColor: Theme.of(context).splashColor,
+          borderRadius: BorderRadius.circular(10),
+          onTap: showButtons,
+          child: const Padding(
+            padding: EdgeInsets.all(4),
+            child: ExpBar()
           ),
         ),
-        const SizedBox(width: 16),
-        Expanded(
-          child: InkWell(
-            splashColor: Theme.of(context).splashColor,
-            child: const Padding(
-              padding: EdgeInsets.all(4),
-              child: ExpBar(),
+        IgnorePointer(
+          ignoring: _hidden,
+          child: AnimatedOpacity(
+            opacity: _hidden ? 0 : 1,
+            duration: const Duration(milliseconds: 200),
+            child: Row(
+              children: [
+                const SizedBox(width: 16),
+                PrimaryChip(
+                  label: '+1',
+                  onPressed: () {
+                    showButtons(shown: false);
+                    charService.updateCharacter(
+                      char.copyWith(
+                        stats: char.stats.copyWith(
+                          currentXp: 1 + (char.stats.currentXp),
+                        ),
+                      ),
+                    );
+                  },
+                ),
+                const Expanded(child: SizedBox(width: 0)),
+                PrimaryChip(
+                  label: '',
+                  icon: const Icon(DwIcons.scroll_quill),
+                  onPressed: () {
+                    showButtons(shown: false);
+                    Get.dialog(const EXPDialog());
+                  },
+                ),
+                const SizedBox(width: 16),
+              ],
```

Comment:
Have you thought about using a `popover` instead of hiding the actual bar? We could place controls much more freely in a popover. It can have add and details icon buttons or the chips you created here

Reply:
See `popover` package and `showPopover`

Reply:
No need to import something new, would be what I think.

Reply:
If you think it would be a lot cleaner, perhaps. But this works quite well and the `IgnorePointer(AnimatedOpacity(...))` feels pretty
1. Easy to understand
2. Not a lot of extra code to support

Reply:
The popover package is already used in the app, so it's nothing new. But I also would consider other solutions, maybe even wrapping the chips you added with a rounded bg container or something, might work nicely. 

It's more the "time limit" of the interaction I am worried about, I'm not sure it's too accessible (see my big comment)